rocketpy/simulation/flight.py

```diff
@@ -1768,34 +1768,42 @@ def u_dot_parachute(self, t, u, post_processing=False):
             e0dot, e1dot, e2dot, e3dot, alpha1, alpha2, alpha3].
 
         """
-        # Parachute data
-        cd_s = self.parachute_cd_s
-        ka = 1
-        R = 1.5
-        rho = self.env.density.get_value_opt(u[2])
-        to = 1.2
-        ma = ka * rho * (4 / 3) * np.pi * R**3
-        mp = self.rocket.dry_mass
-        eta = 1
-        Rdot = (6 * R * (1 - eta) / (1.2**6)) * (
-            (1 - eta) * t**5 + eta * (to**3) * (t**2)
-        )
-        Rdot = 0
         # Get relevant state data
-        x, y, z, vx, vy, vz, e0, e1, e2, e3, omega1, omega2, omega3 = u
-        # Get wind data
+        z, vx, vy, vz = u[2:6]
+
+        # Get atmospheric data
+        rho = self.env.density.get_value_opt(z)
         wind_velocity_x = self.env.wind_velocity_x.get_value_opt(z)
         wind_velocity_y = self.env.wind_velocity_y.get_value_opt(z)
-        free_stream_speed = (
-            (wind_velocity_x - vx) ** 2 + (wind_velocity_y - vy) ** 2 + (vz) ** 2
-        ) ** 0.5
+
+        # Get Parachute data
+        cd_s = self.parachute_cd_s
+
+        # Get the mass of the rocket
+        mp = self.rocket.dry_mass
+
+        # Define constants
+        ka = 1  # Added mass coefficient (depends on parachute's porosity)
+        R = 1.5  # Parachute radius
+        # to = 1.2
+        # eta = 1
+        # Rdot = (6 * R * (1 - eta) / (1.2**6)) * (
+        #     (1 - eta) * t**5 + eta * (to**3) * (t**2)
+        # )
+        # Rdot = 0
```

Comment:
If these comments are important to understand the code or a legacy implementation, this should mentioned. As it is now, it seems to me an unfinished code block.

Reply:
Honestly I don't know. Could be something unfinished or not. 

Old code was not making sense: you calculate Rdot, then u assign Rdot=0.

What I did was just comment it so we don't waste time on these operations.

I wouldn't delete it because I see that we could use it in the future.

Reply:
Let's keep it as it is.